tests/unit/components/NcRelatedResourcesPanel/NcRelatedResourcesPanel.spec.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * @copyright Copyright (c) 2023 Ferdinand Thiessen <opensource@fthiessen.de>
+ *
+ * @author Ferdinand Thiessen <opensource@fthiessen.de>
+ *
```

Comment:
This is supposed to be your contacts, not Ferdinand's :)

Reply:
See: https://github.com/nextcloud/server/blob/master/contribute/HowToApplyALicense.md